_docs/concepts/traffic-management/overview.md

```diff
@@ -1,41 +1,59 @@
 ---
 title: Overview
-overview: Provides a conceptual overview of traffic management principles in Istio and the kind of features enabled by these principles.
+overview: Provides a conceptual overview of traffic management in Istio and the features it enables.
                 
 order: 0
 
 layout: docs
 type: markdown
 ---
 
-Existing container orchestration platforms like Kubernetes, and Mesos, and
-other microservice frameworks allow operators to control when a particular
-set of pods/VMs should receive traffic (e.g., by adding/removing specific
-labels). Unlike existing techniques, Istio decouples traffic flow and infrastructure
-scaling. 
+This page provides an overview of how traffic management works
+in Istio, including the benefits of its traffic management
+principles. It assumes that you've already read [What Is
+Istio?]({{home}}/docs/concepts/what-is-istio/overview.html) and are familiar
```

Comment:
I think the link needs to all be on one line for markdown to be processed correctly.